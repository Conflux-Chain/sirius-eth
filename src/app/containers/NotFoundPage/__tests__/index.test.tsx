import React from 'react';
import { render } from '@testing-library/react';
import { NotFoundPage } from '..';

const renderComponent = () => render(<NotFoundPage />);

describe('<NotFoundPage />', () => {
  it('should match the snapshot', () => {
    const component = renderComponent();
    expect(component.container.firstChild).toMatchSnapshot();
  });
});
