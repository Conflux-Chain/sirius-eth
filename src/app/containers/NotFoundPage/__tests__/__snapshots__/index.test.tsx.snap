// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should match the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: absolute;
  height: calc(100% - 8rem);
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background: #f5f6fa;
  width: 100%;
}

.c1 {
  margin-right: 7rem;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c3 {
  display: inline-block;
  font-size: 1.5714rem;
  line-height: 2rem;
  color: #424242;
  font-weight: 500;
  margin-bottom: 1rem;
}

.c4 {
  display: inline-block;
  color: #4b4b4b;
  opacity: 0.4;
  font-weight: 500;
  line-height: 1.2857rem;
  margin-bottom: 1rem;
  max-width: 540px;
}

.c5 {
  width: 15.7143rem;
  height: 3.5714rem;
  background-color: #fff;
  border-radius: 2.8571rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  color: #545454;
  font-size: 1.1429rem;
  margin-top: 2rem;
}

@media (max-width:600px) {
  .c0 {
    height: calc(100% - 116px);
    width: calc(100% - 32px);
    -webkit-align-items: inherit;
    -webkit-box-align: inherit;
    -ms-flex-align: inherit;
    align-items: inherit;
    -webkit-align-content: center;
    -ms-flex-line-pack: center;
    align-content: center;
  }
}

@media (max-width:600px) {
  .c1 {
    margin-right: 0;
    max-width: 80%;
  }
}

@media (max-width:1024px) {
  .c2 {
    padding: 1rem;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    text-align: center;
  }
}

<div
  class="c0"
>
  <img
    alt="404"
    class="c1"
    src="404.svg"
  />
  <div
    class="c2"
  >
    <span
      class="c3"
    >
      Thereâ€™s nothing here!
    </span>
    <span
      class="c4"
    >
      Sorry! This is an invalid search string.
    </span>
    <span
      class="c4"
    >
      If you're searching for Conflux addresses, please search for a valid address format (hex address or a checksummed-base32 address).
    </span>
    <a
      class="c5"
      href="/"
    >
      Go TO HOMEPAGE
    </a>
  </div>
</div>
`;
